docs/reference/framework-components/framework-components.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Framework Components"
+sidebar_label: "Framework Components"
+sidebar_position: 1
+keywords:
+  - types
+  - conversion
+draft: false
+---
+
+A high level description of the set of built in framework components.
```

Comment:
This is a good page to have. We're calling this Darklang Cloud, and having a separation between the concept of the language and the cloud components (vs eg the cli). Since we're also doing a CLI, might be good to have a parallel page for that.

Reply:
Ok. I've broken this up. 

Is there any place I can read about the concepts of the CLI and cloud framework?

Reply:
https://www.notion.so/darklang/2023-plan-4d065163234e4c8e8c11a786f2624c7c
https://www.notion.so/darklang/Project-Initial-Release-plan-a5dcbeacd0864c06b64da36a1818c537
https://www.notion.so/darklang/Design-How-to-write-ship-code-dafc6669effd44b18019b0439506d0de
https://www.notion.so/darklang/Design-Deployment-model-f32236d7fa6e4a6aa86f3c57cd050b98

Some decent content got shipped to the homepage which includes some of this, but the drafts might also be helpful to you:
https://www.notion.so/darklang/done-Project-New-homepage-Oct-2023-a9ac6f09c4a54888b2cc82b3cbaebc7d
https://www.notion.so/darklang/New-homepage-iteration-3-dec-2-2735959a4f344e34a86ed7be500c201f